packages/components/src/components/data-list/data-list-row.jsx

```diff
@@ -1,8 +1,9 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import MobileDescription from './mobile-description.jsx';
 
-const DataListRow = ({ action_desc, destination_link, row_gap, row_key, rowRenderer, ...other_props }) => {
+const DataListRow = ({ action_desc, desc_titles, destination_link, row_gap, row_key, rowRenderer, ...other_props }) => {
```

Comment:
Nit: Help to add `desc_title` to `DataListRow.propTypes`

Reply:
Okay 👍